examples/training/object_detection/pascal_voc/faster_rcnn.py

```diff
@@ -289,4 +173,4 @@ def pad_fn(examples):
     rpn_box_loss="Huber",
     rpn_classification_loss="BinaryCrossentropy",
 )
-model.fit(train_ds, epochs=18, validation_data=eval_ds, callbacks=callbacks)
```

Comment:
why remove validation_data? I think they're here to calculate `val_loss`

Reply:
Updated and rerverted to track val_loss